src/components/form-input.tsx

```diff
@@ -0,0 +1,18 @@
+type FormInputProps = {
+  input: React.InputHTMLAttributes<HTMLInputElement>;
+  error?: string;
+};
+
+export default function FormInput({ input, error }: FormInputProps) {
+
+  return (
+    <>
+      {input}
+      {error && (
+        <span className="ml-[5px] mt-[5px] text-[13px] leading-[18px] text-red-500">
+          {error}
+        </span>
+      )}
+    </>
+  );
+}
```

Comment:
```suggestion
export default function FormInput({ input, error }: FormInputProps) {

  return (
    <>
     <Input
                       {...props}
                      />
      {error && (
        <span className="ml-[5px] mt-[5px] text-[13px] leading-[18px] text-red-500">
          {error}
        </span>
      )}
    </>
  );
}
```

đã form input thì đưa input vào đây.
chứ truyền vào làm gì thế em